sources/bloodhound/src/output.rs

```diff
@@ -43,7 +43,13 @@ impl ReportWriter for TextReportWriter {
         writeln!(output, "{:17}{}", "Skipped:", report.skipped)?;
         writeln!(output, "{:17}{}", "Total checks:", report.total)?;
         writeln!(output)?;
-        writeln!(output, "Compliance check result: {}", report.status)
+        if report.contain_known_fail_check("3.4.1.1".to_string()) {
+            writeln!(
+                output,
+                "\x1b[93m WARNING: For Kubernetes Variants, DROP will be unconditionally overwritten. If this applies to you, work with your auditor for an exception. See https://github.com/bottlerocket-os/bottlerocket-core-kit/issues/540 for more details.\x1b[0m"
+            )?;
+        }
```

Comment:
I'd prefer a more generic mechanism where we can drop in explanations like this on a per-variant basis, and have them appended to the output. This is really an `aws-k8s-*` specific warning and shouldn't be shown elsewhere.

For example, we could add an optional JSON metadata file alongside the files here:
```
ls -1 /usr/libexec/cis-checks/bottlerocket/
br01010101
br01020100
...
br03040101
br03040101.json <- new
...
```

Or, since the JSON files aren't executable, they could live in `/usr/share`:
```
ls -1 /usr/share/cis-checks/bottlerocket/
br03040101.json
```

And then within the JSON file, we could have different explanation strings for "failed", "skipped", or even "passed".